skycoin/skywire
# remove updater

Did you run `make format && make check`?
yes

An alternative to PR[#1241](https://github.com/skycoin/skywire/pull/1241)

 Changes:	
-	remove all code associated with the updater

The proposed alternative to the updater is to open a dmsgpty terminal and install skywire using the wiki documentation for [package installation](https://github.com/skycoin/skywire/wiki/Skywire-Package-Installation)

The skybian package provides the apt repository configuration as well as the repository signing public key; an amd64 skybian package is now available.

How to test:
```
git clone https://github.com/the-skycoin-project/skywire

git checkout remove-updater-1

sudo make run-source
```
* open dmsgpty terminal
* follow the [package installation wiki](https://github.com/skycoin/skywire/wiki/Skywire-Package-Installation) to install skywire as a package

The commands as referenced in the documentation linked above, for updating or installing skywire via the dmsgpty terminal:

```
curl https://deb.skywire.skycoin.com/archive/skybian-$(dpkg --print-architecture).deb -o skybian-$(dpkg --print-architecture).deb

dpkg -i skybian-*.deb

rm skybian-*.deb

apt update

export DMSGPTYTERM=1

apt install skywire-bin
```

the user is then prompted to start or restart the service.

Refreshing the hypervisor interface should allow one to log into the hypervisor of the newly updated installation.
